modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -0,0 +1,69 @@
+//tag::connectedSelect[]
+
+== Select Ciphers
+
+When you configure a TLS context, Secrets Manager applies default ciphers based on the TLS versions you select. In addition to the defaults, you can select other ciphers to use with the selected TLS version. Each TLS context can have multiple ciphers.
+
+To select ciphers:
+
+. Click *Ciphers* to see available ciphers. 
+. Select ciphers.
++
+If you select a *TLS Version* value of *TLS 1.3*, the default selection is not customizable.
```

Comment:
TLS 1.3 cipher suite list is not even displayed in the UI. This is on purpose as it may vary according to the version of Flex that this TLS context will be applied to.